AmazonChimeSDK/AmazonChimeSDKTests/internal/audio/DefaultAudioClientControllerTests.swift

```diff
@@ -140,7 +141,92 @@ class DefaultAudioClientControllerTests: CommonTestCase {
                                             sessionToken: self.joinToken,
                                             audioWsUrl: self.audioFallbackUrl,
                                             callKitEnabled: false,
-                                            appInfo: any())).wasCalled()
+                                            appInfo: any(),
+                                            audioMode: .Stereo48K)).wasCalled()
+        verify(eventAnalyticsControllerMock.publishEvent(name: .meetingStartRequested)).wasCalled()
+        XCTAssertEqual(.started, DefaultAudioClientController.state)
+        verify(audioLockMock.unlock()).wasCalled()
+    }
+
+    func testStartWithMono48K_startedOk() {
+        DefaultAudioClientController.state = .initialized
+        given(audioSessionMock.getRecordPermission()).willReturn(.granted)
+        given(audioClientMock.startSession(any(),
+                                           basePort: any(),
+                                           callId: any(),
+                                           profileId: any(),
+                                           microphoneMute: any(),
+                                           speakerMute: any(),
+                                           isPresenter: any(),
+                                           sessionToken: any(),
+                                           audioWsUrl: any(),
+                                           callKitEnabled: any(),
+                                           appInfo: any(),
+                                           audioMode: any())).willReturn(AUDIO_CLIENT_OK)
```

Comment:
Can we have this given statement in setup method?

Reply:
Will do